charts/federation-v2/charts/controllermanager/templates/federationconfig.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: core.federation.k8s.io/v1alpha1
+kind: FederationConfig
+metadata:
+  name: federation-v2
+  namespace: {{ .Release.Namespace }}
+spec:
+  limited-scope: {{ .Values.global.limitedScope }}
+{{- if .Values.registryNamespace }}
+  registry-namespace: {{ .Values.registryNamespace }}
+{{- else }}
+{{- if .Values.global.limitedScope }}
+  registry-namespace: {{ .Release.Namespace }}
+{{- else }}
+  registry-namespace: kube-multicluster-public
+{{- end }}
+{{- end }}
+  controller-duration:
+{{- if .Values.clusterAvailableDelay }}
+    available-delay: {{ .Values.clusterAvailableDelay }}
+{{- else }}
+    available-delay: 20s
+{{- end }}
+{{- if .Values.clusterUnavailableDelay }}
+    unavailable-delay: {{ .Values.clusterUnavailableDelay }}
+{{- else }}
+    unavailable-delay: 60s
+{{- end }}
+{{- if .Values.clusterMonitorPeriod }}
+    cluster-monitor-period: {{ .Values.clusterMonitorPeriod }}
+{{- else }}
+    cluster-monitor-period: 40s
+{{- end }}
+  leader-elect:
+{{- if .Values.leaderElectLeaseDuration }}
+    lease-duration: {{ .Values.leaderElectLeaseDuration }}
+{{- else }}
+    lease-duration: 15s
+{{- end }}
+{{- if .Values.leaderElectRenewDeadline }}
+    renew-deadline: {{ .Values.leaderElectRenewDeadline }}
+{{- else }}
+    renew-deadline: 10s
+{{- end }}
+{{- if .Values.leaderElectRetryPeriod }}
+    retry-period: {{ .Values.leaderElectRetryPeriod }}
+{{- else }}
+    retry-period: 5s
+{{- end }}
+{{- if .Values.leaderElectResourceLock }}
+    resource-lock: {{ .Values.leaderElectResourceLock }}
+{{- else }}
+    resource-lock: configmaps
+{{- end }}
+  feature-gates:
```

Comment:
Should it be possible to configure the feature gate to enable?

Reply:
done. 

I did not do it because we always enable all features. However, make the feature complete, I add the feature-gates in helm support.

Reply:
By the way, I organize `feature-gates` in nested way because I think it makes sense in `feature-gates` case.